### PR TITLE
TD-11061:migrate datax.py to python3, and compatible with python2

### PR DESCRIPTION
**Pull request by dingbo8128**

### CODE REVIEW

**Comment by sangshuduo** [comment]
How about submitting this PR back to the original DataX repo too? Then we don't need to maintain a diversity code if the DataX maintainer accepts it.

**Review by sangshuduo — APPROVED** [review]
LGTM

**Comment by dingbo8128** [comment]
> 



> How about submitting this PR back to the original DataX repo too? Then we don't need to maintain a diversity code if the DataX maintainer accepts it.

But, how to submit to the original repo after merged and closed ?

**Pull request merged**
